src/content/1.7/modules/concepts/hooks/list-of-hooks.md

```diff
@@ -1060,6 +1060,38 @@ actionWatermark
     );
     ```
     
+actionGetAdminOrderButtons
+: 
+    This hook is used to generate the buttons collection on the order view page thanks (see ActionsBarButtonsCollection)
```

Comment:
```suggestion
    This hook is used to generate the buttons collection on the order view page (see ActionsBarButtonsCollection)
```

Reply:
Damn, this means the core needs to be fixes as well 😅